projects/admin-core/src/lib/search-index/search-index-scheduling/search-index-scheduling.component.ts

```diff
@@ -0,0 +1,120 @@
+import { Component, OnInit, ChangeDetectionStrategy, Output, EventEmitter, DestroyRef, Input } from '@angular/core';
+import { FormControl, FormGroup } from '@angular/forms';
+import { SearchIndexModel, TaskSchedule } from '@tailormap-admin/admin-api';
+import { takeUntilDestroyed } from '@angular/core/rxjs-interop';
+import { debounceTime, distinctUntilChanged, filter, map } from 'rxjs';
+import { FormHelper } from '../../helpers/form.helper';
+import { MatSelectChange } from '@angular/material/select';
+
+@Component({
+  selector: 'tm-admin-search-index-scheduling',
+  templateUrl: './search-index-scheduling.component.html',
+  styleUrls: ['./search-index-scheduling.component.css'],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class SearchIndexSchedulingComponent implements OnInit {
+
+  public taskSchedule: TaskSchedule | undefined = undefined;
+
+  @Input({ required: true })
+  public set searchIndex(form: SearchIndexModel | null) {
+    this.taskSchedule = form?.schedule;
+    this.initForm(form?.schedule);
+  }
+
+  @Output()
+  public updateSchedule = new EventEmitter<{ searchIndex: Pick<SearchIndexModel, 'schedule'> }>();
+
+  @Output()
+  public formChanged = new EventEmitter<boolean>();
+
+  public scheduleOptions = [
+    { cronExpression: '', viewValue: $localize `:@@admin-core.search-index.schedule.no-schedule:No schedule` },
+    { cronExpression: '0 0 0/1 1/1 * ? *', viewValue: $localize `:@@admin-core.search-index.schedule.every-hour:Every hour` },
+    { cronExpression: '0 0 18 1/1 * ? *', viewValue: $localize `:@@admin-core.search-index.schedule.every-day:Every day at 18:00` },
```

Comment:
Maybe do the default jobs at night? Around 03:00 or so? @mprins what do you think?